weston-ivi-shell/src/ivi-controller.c

```diff
@@ -1820,6 +1833,9 @@ surface_event_create(struct wl_listener *listener, void *data)
            (struct ivi_layout_surface *) data;
     uint32_t id_surface = 0;
 
+    if (is_desktop_surface_create(shell, layout_surface))
+        wl_signal_emit(&shell->desktop_surface_created_signal, layout_surface);
```

Comment:
As I see, there is no signal name `desktop_surface_created_signal` in Weston. for deskto_surface, they are using the callback instead of the signal. 
Anyway, you are right, the name should shows correctly its idea, the `id_allocation_request_signal` fine with me